clarifai/runners/dockerfile_template/Dockerfile.cuda.template

```diff
@@ -149,14 +118,11 @@ WORKDIR /app
 COPY . /app/model_dir/${name}
 
 # Copy the checkpoints directory into the distroless image.
-COPY --from=build /app/checkpoints /app/model_dir/${name}/1/checkpoints
```

Comment:
it wouldn't be from=build right?

Reply:
oh right, just corrected it

Reply:
I just realized that we don't need to add the COPY checkpoints line in the Dockerfile, since we're already using `COPY . /app/model_dir/${name}` a line before. That line will also copy the checkpoint directory into the final folder of the container. What do you think?

Reply:
we might want to separate out the code from the checkpoint though in the future as copying in the code after the checkpoint would re-use most layers if you'd iterating on the same model code a few times hopefully it can re-use the checkpoint layer.